build/windows/installer/conf/fluent-bit.conf

```diff
@@ -45,5 +45,18 @@
     Match oms.container.log.la.*
     Exclude stream ${AZMON_LOG_EXCLUSION_REGEX_PATTERN}
 
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               oms.container.log.la.*
```

Comment:
Add the  alias "#${KubernetesFilterEnabled} Alias oms_kubernetes_filter"  after Name . This will be useful for the metrics and troubleshooting.

Reply:
added